content/topics/intro-to-css-architecture/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Intro to CSS architecture
+ready: true
+---
+
+## Pre-workshop readings
```

Comment:
Remove the words "Pre-workshop readings"

Don't just resolve conversations if you don't know what they mean. Rather ask for clarification.